### PR TITLE
[325] Amélioration page de login

### PR DESCRIPTION
**Pull request by jdauphant**
image
image
image

### CODE REVIEW

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=4735123&bid=15807849)

**Comment by jdauphant** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aplus/aplus/pullRequest?prid=4735123&bid=15807849)

File: app/views/loginHome.scala.html [hunk 1]
```diff
@@ -65,30 +68,40 @@
...
                         }
-                        <button type="submit" class="mdl-button mdl-js-button mdl-button--raised" name="id" disabled>Connexion</button>
+                        <button type="submit" class="mdl-button mdl-js-button mdl-button--raised" name="id" disabled>Envoyer</button>
                     </form>
                     }
                     case Left(Some(user)) => {
-                        <h3><i class="material-icons">mail_outline</i> Regardez vos emails <i class="material-icons">mail_outline</i></h3>
-                         <p>Vous allez recevoir un email à l'adresse <b>@user.email</b>. Cliquez sur le lien contenu dans l'email pour accéder à Administration+.</p>
+                        <h3><i class="material-icons">mail_outline</i><br>Consultez vos emails</h3>
+                         <p>Nous avons envoyé par e-mal un lien valable 15 minutes à l'adresse <b>@user.email</b>. Cliquez sur le lien contenu dans l'email pour accéder à Administration+.</p>
```
Comment (lpereir4) [inline app/views/loginHome.scala.html]: correction typographique : e-mal => e-mail

File: app/views/loginHome.scala.html [hunk 2]
```diff
@@ -110,8 +123,8 @@ <h3>Vous allez accéder automatiquement à Administration+</h3>
                         </script>
                     }
                 }
-                <br>
-                <p>Vous rencontrez des difficultés à vous connecter ? Contactez-nous sur <a href="mailto:@Constants.supportEmail?subject=Problème%20A+%20Connexion">@Constants.supportEmail</a></p>
+                <br><br>
+                <p style="margin-top: 30px">Vous rencontrez des difficultés à vous connecter ? Contactez-nous sur <a href="mailto:@Constants.supportEmail?subject=Problème%20A+%20Connexion">@Constants.supportEmail</a></p>
```
Comment (lpereir4) [inline app/views/loginHome.scala.html]: Faut-il créer une classe css ?

Comment (lpereir4) [reply app/views/loginHome.scala.html]: De plus, si nous utilisons des marges, autant supprimer les br.

File: app/views/loginHome.scala.html (hunk 1 above)
Comment (jdauphant) [reply app/views/loginHome.scala.html]: J'avais corrigé la typo la semaine dernière ;)

**Pull request merged**